sonar-orchestrator/src/main/java/com/sonar/orchestrator/container/SonarDistribution.java

```diff
@@ -121,11 +123,25 @@ public boolean isRelease() {
     return version.isRelease();
   }
 
+  /**
+   * @deprecated since 3.15
+   */
+  @Deprecated
   public SonarDistribution activateLicense(String pluginKey) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)